awpy/analytics/stats.py

```diff
@@ -620,6 +621,21 @@ def _handle_multi_kills(
             _increment_statistic(player_statistics, player, n_kills)
 
 
+def _handle_mvps(
+    game_round: GameRound,
+    player_statistics: dict[str, PlayerStatistics],
+    round_statistics: RoundStatistics,
+) -> None:
+    if "mvpName" in game_round:
```

Comment:
That check should be unnecessary since we per type definition a "GameRound" should always have that key.

But it probably does make sense to leave this in to allow the stat extraction of old json files that were produced with a previous version.